src/components/RangeFilm/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Range } from 'rc-slider'
+import { FILM_ADVERTISING } from '../../constans'
+
+const { MIN } = FILM_ADVERTISING
+
+const RangeFilm = ({
+  field,
+  setValueFilm,
+  value,
+  max,
+  marksMin,
+  marksMax,
+  marksMinLabel,
+  marksMaxLabel,
+}) => {
+  return (
+    <Range
+      {...field}
+      onChange={(value) => {
+        setValueFilm(value)
+        field.onChange(value)
+      }}
```

Comment:
you had almost empty function body
except for return statement
so you can simplify JSX by separating props to variables and use `useMemo` and `useCallback`

```
const handleChange = (value) => {
        setValueFilm(value)
        field.onChange(value)
}}

...

onChange={handleChange}
```